### PR TITLE
feat(YouTube): Channel as largeImageKey

### PR DESCRIPTION
**Pull request by down-bad**
Merry Christmas

Choose between YT logo, thumbnail or channel image
image

Option to show channel image when viewing a channel
image

Use url for the YT logo so it shows correctly on mobile
Old: image

New: image

### CODE REVIEW

File: websites/Y/YouTube/presence.ts [hunk 1]
```diff
@@ -287,7 +298,8 @@ presence.on("UpdateData", async () => {
       else presenceData.details = (await strings).watchVid;
 
       delete presenceData.state;
-      presenceData.largeImageKey = "yt_lg";
+      presenceData.largeImageKey =
+        "https://cdn.discordapp.com/app-assets/463097721130188830/513734690272968717.png?size=1024";
```
Comment (Bas950) [inline websites/Y/YouTube/presence.ts:302]: Revert, this will rate limit hard.

File: websites/Y/YouTube/presence.ts [hunk 2]
```diff
@@ -324,7 +336,8 @@ presence.on("UpdateData", async () => {
     document.location.hostname === "youtube.com"
   ) {
     const presenceData: PresenceData = {
-        largeImageKey: "yt_lg"
+        largeImageKey:
+          "https://cdn.discordapp.com/app-assets/463097721130188830/513734690272968717.png?size=1024"
```
Comment (Bas950) [inline websites/Y/YouTube/presence.ts:340]: Revert, this will rate limit hard.

File: websites/Y/YouTube/presence.ts [hunk 3]
```diff
@@ -510,7 +530,8 @@ presence.on("UpdateData", async () => {
     else presence.setActivity(presenceData);
   } else if (document.location.hostname === "studio.youtube.com") {
     const presenceData: PresenceData = {
-        largeImageKey: "yt_lg",
+        largeImageKey:
+          "https://cdn.discordapp.com/app-assets/463097721130188830/513734690272968717.png?size=1024",
```
Comment (Bas950) [inline websites/Y/YouTube/presence.ts:534]: Revert, this will rate limit hard.

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]
lgtm

**Pull request merged**